decidim-core/app/assets/javascripts/decidim/input_multiple_mentions.js.es6

```diff
@@ -0,0 +1,176 @@
+      }
+      // Set recipient profile view
+      let recipientLabel = `
+        <label style="padding: 0 0 10px 0" onClick="deleteRecipient(this)">
+          <img src="${item.original.avatarUrl}" alt="${item.original.name}" height="35" width="35" style="border-radius: 50%;">&nbsp;
+          <b>${item.original.name}</b>
+          <input type="hidden" name="recipient_id[]" value="${item.original.id}">
+        </label>
+      `;
+      // Append new recipient to DOM
+      if (item.original.disabledNotifications === "") {
+        $multipleMentionRecipientsContainer.append(recipientLabel);
+        $multipleMentionContainer.val("");
+      }
+      // Clean input
+      return "";
+    },
+    menuItemTemplate: function(item) {
+      let disabledElementClass = ((item.original.disabledNotifications === "") ? "" : "class=\"disabled-notifications\"")
+      let tpl = `<div ${disabledElementClass}><strong>${item.original.nickname}</strong>&nbsp;<small>${item.original.name}</small>&nbsp;<span class="disabled-notifications-info">${item.original.disabledNotifications}</span></div>`;
+      return tpl;
+    }
+  });
+
+  let setupEvents = function($element) {
+    // DOM manipulation
+    $element.on("focusin", (event) => {
+      // Set the parent container relative to the current element
+      tribute.menuContainer = event.target.parentNode;
+    });
+    $element.on("focusout", (event) => {
+      let $parent = $(event.target).parent();
+
+      if ($parent.hasClass("is-active")) {
+        $parent.removeClass("is-active");
+      }
+    });
+    $element.on("input", (event) => {
+      let $parent = $(event.target).parent();
+
+      if (tribute.isActive) {
+        // We need to move the container to the wrapper selected
+        let $tribute = $(".tribute-container");
+        $tribute.appendTo($parent);
+        // // Remove the inline styles, relative to absolute positioning
+        $tribute.removeAttr("style");
+        // Parent adaptation
+        $parent.addClass("is-active");
+      } else {
+        $parent.removeClass("is-active");
+      }
+    });
+  };
+
+  // Call only if we have containter to bind events to
+  if ($multipleMentionContainer.length) {
+    setupEvents($multipleMentionContainer);
+  }
+
+  // This allows external libraries (like React) to use the component
+  // by simply firing and event targeting the element where to attach Tribute
+  $(document).on("attach-mentions-element", (event, element) => {
+    if (!element) {
+      return;
+    }
+    tribute.attach(element);
+    // Due a bug in Tribute, re-add menu to DOM if it has been removed
+    // See https://github.com/zurb/tribute/issues/140
+    if (tribute.menu && !document.body.contains(tribute.menu)) {
+      tribute.range.getDocument().body.appendChild(tribute.menu);
+    }
+    setupEvents($(element));
+  });
+
+  // tribute.attach($multipleMentionContainer);
+  // Tribute needs to be attached to the `.ql-editor` element as said at:
```

Comment:
Overall, I'd remove all references to a `.ql-editor`, and all the code related to it. A don't think we never going to use this in the Quill editor. In any case, when we need it, we can refactor this. We should keep this file as simple as possible.

Reply:
Done!